variants/lopy/pins_arduino.h

```diff
@@ -17,7 +17,7 @@
 #define LORA_MOSI 27    // GPIO27 - SX1276 MOSI
 #define LORA_CS 17      // GPIO17 - SX1276 CS
 #define LORA_RST 18     // GPIO18 - SX1276 RST
-#define LORA_IO0 23     // GPIO23 - SX1276 IO0
+#define LORA_IRQ 23     // GPIO23 - SX1276 IO0
```

Comment:
maybe you can add a `#define LORA_IO0 LORA_IRQ` in case someone uses the IO0 definition? Same for the other boards.